.github/workflows/maestro_tests.yml

```diff
@@ -70,6 +70,6 @@ jobs:
             -PmanifestEndpoint=${{ secrets.MANIFEST_ENDPOINT }}
             :app:installStagingRelease
             
-            maestro test maestroUiFlows/login_flow.yaml
+            ./gradlew maestroTests
```

Comment:
~Is there a benefit of running via Gradle instead?~ NM, for all future tests. Got it